resolwe/flow/models/entity.py

```diff
@@ -48,7 +49,7 @@ def __str__(self):
 class Relation(BaseModel):
     """Relations between entities."""
```

Comment:
Include the model documentation in the reference. We need to start documenting out DB models.